ref/src/main/java/io/smallrye/common/ref/References.java

```diff
@@ -32,11 +31,8 @@ static ReferenceQueue<Object> getReaperQueue() {
             if (isBuildTime()) {
                 // do nothing (class should be reinitialized)
             } else {
-                final AtomicInteger cnt = new AtomicInteger(1);
-                final PrivilegedAction<Void> action = () -> startThreadAction(cnt.getAndIncrement());
-                for (int i = 0; i < 3; i++) {
-                    doPrivileged(action);
-                }
+                final PrivilegedAction<Void> action = () -> startThreadAction(1);
```

Comment:
@gastaldi you can use an anonymous class, if the code style is not terribly affected, wdyt @dmlloyd @Sanne ?

Reply:
Would that change anything performance-wise? I can change it if necessary

Reply:
I doubt it would make a difference; this code is only run once at startup. We still haven't shown that there is a measurable difference.

Reply:
Good points!